### PR TITLE
background_selector improvements

### LINKED ISSUE #57: background_selector improvements
Hi to eveybody,
i wanna guess an improvement for the script: "background selector".
I was looking for a solution to improve the background selector script to change it on something more similar to that one is used by google maps or esri etc.
Cattura2

This solution is more suitable to the other one when you have multiple baselayer to change, and you cannot choice from a sort of list but you must go haed until you find the right one. Can someone do this kind of improvement?
i can't just because i'm not a web programmer, i've tried to find a solution without success.
I hope this could help others like me that need this feature,
Best reguards,
Marco

### PR DESCRIPTION
**Pull request by npltr62**
Fixes #57

### CODE REVIEW

**Comment by Gustry** [comment]
Thanks for your contribution, very appreciated.

It seems it's also fix #69 so it's for LWC 3.7 only, right ?

**Comment by Gustry** [comment]
As it seems targeting 3.7 minimum, I think there should be a hard copy of the "master" file, like `background_selector-3-6.js`, what do you think ?

File: library/ui/background_selector/background_selector.js [hunk 1]
```diff
@@ -37,16 +32,20 @@ lizMap.events.on({
...
+                nextbl = $('lizmap-base-layers select option:first');
             var nextbl_text = nextbl.text();
             var nextbl_val = nextbl.attr('value');
             var cur_image_url = $('#baselayer-image-selector').css('background-image');
             var cur_image_spl = cur_image_url.split('/');
-            cur_image_spl.pop();
+            var cur_image = cur_image_spl.pop();
             var nextbl_image = cur_image_spl.join('/') + '/' + nextbl_val + '.png';
-
+            var begin = window.location.protocol + "//" + window.location.host + "/";
```
Comment (nworr) [inline library/ui/background_selector/background_selector.js:44]: What's the aim of this part ? (using relative path i guess) ?

the full path works for me

**Review by nworr — COMMENTED** [review]
Ok for me for 3.7 only

pending question about the backgound image part

File: library/ui/background_selector/background_selector.js (hunk 1 above)
Comment (nworr) [reply library/ui/background_selector/background_selector.js]: this part was removed by  https://github.com/3liz/lizmap-javascript-scripts/commit/0cb93915d923996e8486e919bf82459708694cd1

**Comment by nworr** [comment]
I add some comments and the legace script for 3.6

**Review by Gustry — APPROVED** [review]
Thanks @npltr62 and @nworr

**Pull request merged**